README.md

```diff
@@ -302,6 +302,37 @@ fetcher
 
 For requests from the server, the config object is simply passed into the service being called.
 
+## Using Context Picker
+
+By Default, fetchr appends all context values to the xhr url, `contextPicker` allows you to determine how do you want to pick the context value. They should be the `predicate` function of [lodash/object/pick](https://lodash.com/docs#pick) with three arguments: `(value, key, object)`.
```

Comment:
By Default, fetchr appends all context values to the xhr url as query params. `contextPicker` allows you to greater control over which context variables get sent as query params depending on the xhr method (`GET` or `POST`). This could be useful when you want to limit the number of variables in a `GET` url in order not to accidentally [cache bust](http://webassets.readthedocs.org/en/latest/expiring.html).

`contextPicker` follows the same format as the `predicate` parameter in [`lodash/object/pick`](https://lodash.com/docs#pick) with three arguments: `(value, key, object)`.